lib/components/collapse.vue

```diff
@@ -78,16 +79,22 @@
             },
             clearHeight(el) {
                 el.style.height = null;
+            },
+            emitState() {
+                this.$root.$emit('collapse::toggle::state', this.id, this.state);
             }
         },
-
         created() {
             this.$root.$on('collapse::toggle', target => {
                 if (target !== this.id) {
                     return;
                 }
                 this.toggle();
             });
+            this.emitState();
```

Comment:
Just in case the receiving components that are already in document need to reflect the state (although it does default to false).